uint8 drone_id
uint16 command
uint8 result
string result_text